lms/djangoapps/hera/admin.py

```diff
@@ -13,7 +13,29 @@ class OnboardingAdmin(admin.ModelAdmin):
     """
     Reflection of the hera model on the admin panel.
     """
-    pass
+
+    def get_queryset(self, request):
+        qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        if not qs:
+            default_onboarding = self.model()
+            default_onboarding.save()
+
+            qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        return qs
+
+    def has_add_permission(self, request):
```

Comment:
Need to add docstring.
For a reviewer not obvious how `queryset.count` relates with `has_add_permissios` flag.

Reply:
Done.